pages/test_analytics/flaky_test_assignment.md

```diff
@@ -34,3 +34,8 @@ When an assigned test has not flaked in more than 7 days, it is moved to the **O
 
 <%= image "outdated-assignments.png", width: 1960/2, height: 630/2, alt: "Flaky test page showing team assignments" %>
 
+## Weekly flaky test summary
+
+You're able to schedule a weekly summary of the flakiest tests assigned to your teams. Visit the **Suite settings** page to create new notifications, or manage existing ones. If you would like to set up auto assignment, check out our [Test ownership](/docs/test-analytics/test-ownership) feature.
```

Comment:
Should there be anything about this being available only for pro? Or is this implied from the connection to flaky assignments?

Reply:
This is part of the assignments page which says at the top that it's a paid feature, so I don't think we need to reiterate here